packages/clay-table/src/EditableRow.tsx

```diff
@@ -0,0 +1,142 @@
+/**
+ * © 2019 Liferay, Inc. <https://liferay.com>
+ *
+ * SPDX-License-Identifier: BSD-3-Clause
+ */
+
+import Cell from './Cell';
+import ClayButton from '@clayui/button';
+import React from 'react';
+import Row from './Row';
+import {ClayCheckbox, ClayInput} from '@clayui/form';
+
+interface IEditableCell {
+	cellProps?: Object;
+	editable?: boolean;
+	title: string;
+}
+
+type IRowProps = React.ComponentProps<typeof Row>;
+
+interface IEditableRowProps extends IRowProps {
+	/**
+	 * Defines each Cell in a Row. A cell must have a `title`, can be
+	 * `editable` and can be overrided using `cellProps`.
+	 */
+	cells: Array<IEditableCell>;
+
+	/**
+	 * Callback function for when the selectable checkbox state changes.
+	 */
+	onCheckboxChange?: (isChecked: boolean) => void;
+
+	/**
+	 * Callback function for when the `Save` button is clicked.
+	 */
+	onRowUpdated?: (newEntries: Array<IEditableCell>) => void;
```

Comment:
Maybe `onRowChange`? This creates better consistency with the other API's.

Reply:
Makes sense Matu!